packages/emotion-server/src/stream.js

```diff
@@ -0,0 +1,53 @@
+import through from 'through'
+import tokenize from 'html-tokenize'
+import pipe from 'multipipe'
+import { inserted, registered, names } from 'emotion'
+
+export default function renderStylesToNodeStream() {
+  let insed = {}
+  const tokenStream = tokenize()
+
+  const inlineStream = through(
+    function write(thing) {
```

Comment:
Could this be

Reply:
Yup, it was originally like that but I was debugging something so I changed it.